model/Image.php

```diff
@@ -711,7 +711,7 @@ public function cacheFilename($format) {
 		array_shift($args);
 		$folder = $this->ParentID ? $this->Parent()->Filename : ASSETS_DIR . "/";
 
-		$format = $format . base64_encode(json_encode($args, JSON_NUMERIC_CHECK));
+		$format = $format . base64_encode(json_encode(array_map('strval', $args)));
```

Comment:
Yep, technically it should be but the args are sometimes strings (e.g. FFFFFF for PaddedImage) which would presumably always come out as '1' resulting in collisions